linodecli/configuration/__init__.py

```diff
@@ -242,6 +242,17 @@ def update(
                 continue
             if self.config.has_option(username, key):
                 value = self.config.get(username, key)
+            # different types of database creation use different endpoints,
+            # so we need to set the default engine value based on the type
+            elif key == "engine":
```

Comment:
`engine` hasn't been added to the doc as `x-linode-cli-allowed-defaults` officially. We have a sparate open PR to do this job.
Before https://github.com/linode/linode-api-docs/pull/779 gets merged and live, we can test the change by hardcoding `engine` in the `allowed_defaults` (L234):
`allowed_defaults.append("engine")`

Reply:
> It seems that MongoDB is not available for creation currently, right?

Yep, that's correct 👍

Reply:
Oh thanks for sharing this trick! It's much better than hardcoding the value lol